### DISCUSSION TITLE
No sound on Arch Linux

### DISCUSSION THREAD

**Discussion by Tayergo** [ppy/osu#16123] [Q&A]
Kernel: 5.15.8-arch1-1
OS: ArchLinux
The error I get is "Bass faulted with error code 3: driver"
All of my other sound works fine.
I should note I'm using the .appimage because I'm too lazy to compile.

**Comment by smoogipoo** [comment]
Are you selecting the "Default" device in osu!? Please provide logs https://github.com/ppy/osu/discussions/15188

**Comment by manujedi** [comment]
This happens to me when selecting jack driver with pipewire.
Either pw-jack, the jack pipewire dropin or select alsa/pulse/pipewire as audio in osu settings.
